brigade/static/scss/core/_variables.scss

```diff
@@ -7,16 +7,23 @@ $color-black: #000;
 $color-gray-dark: #5a5a5a;
 $color-white: #fff;
 
+$color-red: #cd1f42;
+$color-light-red: #d5405e;
+$color-dark-red: #bd1b3c;
+$color-blue: #399ed2;
+$color-light-blue: #57acd9;
+$color-dark-blue: #3591c0;
+
 $color-gray: #999;
 $color-gray-light: #e5e5e5;
 
-$color-primary: #cf1b41; /* CfA Red */
-$color-primary-light: #d5305e;
-$color-primary-dark: #bd1b3c;
+$color-primary: $color-red;
+$color-primary-light: $color-light-red;
+$color-primary-dark: $color-dark-red;
 
-$color-secondary: #399ed2; /* CfA Blue */
-$color-secondary-light: #57acd9;
-$color-secondary-dark: #3591c0;
+$color-secondary: $color-blue;
+$color-secondary-light: $color-light-blue;
+$color-secondary-dark: $color-dark-blue;
```

Comment:
Is the blue hero background an intentional part of this change?

Reply:
Actually the hero background on the `feature/style-guide-v4` branch is blue. I remember it having a red background in one of your screenshots -- was that change merged? 

The background color of the hero is defined in the v4 style guide like this: 
`.hero { background-color: $color-blue; }`

Reply:
OH LOL, this required a change to the style guide to actually use this variable. I made the change locally inside my `node_modules` directory for testing but forgot to commit it to the style guide 😳 .

Reply:
Oh actually, I just never merged it - https://github.com/codeforamerica/style-guide/pull/14